be/src/service/internal_service.cpp

```diff
@@ -64,12 +63,14 @@ static void thread_context_deleter(void* d) {
 }
 
 template <typename T>
-class NewHttpClosure : public ::google::protobuf::Closure {
+class NewClosure : public ::google::protobuf::Closure {
```

Comment:
rename it to ClosureWithMemtracker better? since new closure is too common...

Reply:
NewClosure is not used for mem tracker, it is used to delete the recreated request when brpc attachment transmits data